grpc.go

```diff
@@ -68,7 +68,7 @@ var emptyResp = &gpb.Empty{}
 
 const onlineCPUMemWaitTime = 100 * time.Millisecond
 
-const onlineCPUMaxTries = 10
+const onlineCPUMaxTries = 30
```

Comment:
Why not extend it to some round and large value? Something like `100` :)

Reply:
I probably agree - unless we have some hard deadline we know we should meet, then presumably we are looking for vcpus 'never coming online', in which case a large timeout costs us nothing apart from in the interesting rare failure case that we are looking for.
To note, I have seen 'slow' machines take >3s to get to the workload - now I don't know where that time was spent (or if it was related to entropy or not, but I think not)... so I like the idea of `100` more than `30` :-)

Reply:
10 seconds? :thinking:  ...  sounds good